flowtorch/parameters/base.py

```diff
@@ -25,14 +25,16 @@ def __init__(
     def forward(
         self,
         x: Optional[torch.Tensor] = None,
+        y: Optional[torch.Tensor] = None,
```

Comment:
This `y` is for the second part of the coupling split, right? I wonder whether we could do this without having an additional argument to Parameters...?